src/auth/responses/generate-token.response.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class GenerateTokenResponse {
+  @ApiProperty({
+    description: "Generate accessToken",
+    example:
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Impob25AZ21haWwuY29tIiwiaWQiOjIsImlhdCI6MTY4ODQ2Mzc4OCwiZXhwIjoxNjg4NTUwMTg4fQ.F3YqRedhg62eXpJ946OOTE52Y5-GIYHC8GTtT8JNMc8",
+    type: "string",
```

Comment:
Why is the type required here? Wouldn't it be automatically detected based on the context?

Reply:
No it can't.